doc/_templates/autosummary/class.rst

```diff
@@ -4,7 +4,7 @@
 
 .. autoclass:: {{ objname }}
 
-{% if attributes %}
+{% if '_aliases' not in attributes %}
```

Comment:
Actually, we might need a better solution, because other class pages still show an empty 'Attributes' section :sweat_smile: E.g. in these places:
- https://pygmt-dev--4131.org.readthedocs.build/en/4131/api/generated/pygmt.config.html
- https://pygmt-dev--4131.org.readthedocs.build/en/4131/api/generated/pygmt.GMTBackendEntrypoint.html#pygmt.GMTBackendEntrypoint

Maybe revert this change, and we'll fix it in a separate PR?

Reply:
Ah, OK.
Yeah, we can handel this later in a separate PR.